packages/components/tabs/README.mdx

```diff
@@ -27,6 +27,14 @@ import { Tabs } from '@contentful/f36-tabs';
 
 ```
 
+### Vertical
+
+Tabs can be rendered with vertical navigation too, keep in mind that dividers variants are ignored for for vertical tabs
```

Comment:
> What's the reason for moving to a controlled tab state?

Radix Tabs doesn't expose which tab is active, so we couldn't pass the isActive to the NavList, but this is still a WIP, until the NavList is merged, so we can change it.
Also kept as orientation since it's the prop used on the Radix Tabs, but maybe we should call it variant like in other components, WDYT?